content/tormobile/tormobile-6/contents.lr

```diff
@@ -0,0 +1,11 @@
+_model: question
+---
+title: Do I need Tor Browser for Android and Orbot, or only one?
+---
+seo_slug: tor-browser-for-android-and-orbot
+---
+description:
+While both Tor Browser for Android and Orbot are great, they serve different purposes, so unless you need both of their functionalities, then you don't need both.
+Tor Browser for Android is like the desktop Tor Browser, but on your phone. It is a one stop mobile browser that uses the Tor network and tries to be as anonymous as possible.
```

Comment:
Might be worth mentioning that TB for Android already has Orbot built in so Orbot isn't required for it to function.

Reply:
I feel like that's already pretty heavily implied. If it needed the Orbot app to function then the answer to this question would be totally different, and the TBfA wouldn't be preforming its most important function.

I suppose there's nothing terribly wrong with being _too_ explicit, but I think this is already implicit enough to the point where mentioning that would be redundant and might even confuse people. Especially because the Orbot in TBfA is hidden from the end-user.

Reply:
Okay, fair enough.

Reply:
> > I feel like that's already pretty heavily implied. If it needed the Orbot app to function then the answer to this question would be totally different, and the TBfA wouldn't be preforming its most important function.
> 
> I think it is fair to mention that TBA has fagocitated Orbot (without using the word fagocitate!), because we give recognition to the work Orbot people put and we are using.
> I don't see that implied from the text. The text feels more like TBA replaced Orbot, but it should feel that TBA benefits from the advances of Orbot because it is incorporated inside.

does fagocitate mean like absorbed?

Reply:
Yeah, seems to be that way based on my internet searching. Comes out of latin/italian, neat.

Reply:
Okay, in [87771d8](https://github.com/torproject/support/pull/33/commits/87771d83bab6fce14ca502a19f508ee72b4a776a) I tried to incorporate this information clearly into the answer, but I can't help but feeling it still got a bit more convoluted than necessary. Maybe you two have ideas to make it less potentially confusing/more elegant?

Reply:
ooops sorry for being pedantic, but you got what i meant at the end!

Reply:
No worries, I like learning new words!